.storybook/util/withState.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React, {type Node, Component} from 'react';
+
+class WithState extends Component<{|initialState: {}, children: Function, refs?: any[]|}, {}> {
+  state = {
+    ...this.props.initialState
+  }
+  componentDidMount() {
+    this.refs = this.props.refs;
```

Comment:
this is firmly in the i'm not sure if it should work but it works' camp, given this is internal only I think that's fine but I'm happy to investigate